crates/net/network/src/config.rs

```diff
@@ -434,6 +392,35 @@ impl NetworkConfigBuilder {
         }
     }
 
+    /// Exposes the discovery v5 config builder.
```

Comment:
this should document was the closure actually does, like this gets only called if the builder is set.